x-pack/test/security_api_integration/tests/features/deprecated_features.ts

```diff
@@ -489,56 +489,88 @@ export default function ({ getService }: FtrProviderContext) {
       const ruleOneTransformed = await createRule(
```

Comment:
__question:__ Forgot to mention during the review: I’m wondering if we want to add a test similar to `'all deprecated features are replaced by a single feature only'`, but specifically for alerting. For instance, if someone deprecates a feature that includes alerting with, say, a single rule type and single consumer, and replaces it with another feature or features that provide access to alerting rules or consumers not registered in `deprecatedFeature.alerting`, we could fail the test. In that case, we could prompt the developer to either update `deprecatedFeature.alerting` or mark a specific case as "expected" in the test.